package.json

```diff
@@ -15,12 +15,17 @@
     "httpntlm": "^1.5.2",
     "request": "^2.60.0",
     "underscore": "^1.8.3",
-    "underscore-deep-extend": "0.0.5",
+    "underscore-deep-extend": "github:kurtmilam/underscoreDeepExtend",
```

Comment:
So the package has been updated and this gets easy again :). I'll add a new commit with the new version.